dev/tasks/r/github.packages.yml

```diff
@@ -303,7 +303,7 @@ jobs:
           - {image: "rhub/debian-clang-devel", libarrow_binary: "TRUE"}
           # fedora-clang-devel cannot use binaries bc of libc++ (uncomment to see the error)
           # - {image: "rhub/fedora-clang-devel", libarrow_binary: "TRUE"}
-          - {image: "rhub/ubuntu-gcc-release"} # currently ubuntu-20.04 (focal)
+          - {image: "rhub/ubuntu-release"} # currently ubuntu-20.04 (focal)

```

Comment:
I wonder if this job and the next two are now redundant. According to https://r-hub.github.io/containers/, ubuntu-release is also on 22.04.

Reply:
Yeah, they probably are. I'll keep just this one

Reply:
I also noticed the old rhub image up above and changed that too.